handler/handler.go

```diff
@@ -384,11 +383,12 @@ func (h *InstanceComplete) UpdateInstance(ctx context.Context, e *event.ExportSt
 // ProduceExportCompleteEvent sends the final kafka message signifying the export complete
 func (h *InstanceComplete) ProduceExportCompleteEvent(e *event.ExportStart, rowCount int32) error {
 	if err := h.producer.Send(schema.CSVCreated, &event.CSVCreated{
-		InstanceID: e.InstanceID,
-		DatasetID:  e.DatasetID,
-		Edition:    e.Edition,
-		Version:    e.Version,
-		RowCount:   rowCount,
+		InstanceID:   e.InstanceID,
+		DatasetID:    e.DatasetID,
+		Edition:      e.Edition,
+		Version:      e.Version,
+		RowCount:     rowCount,
+		DimensionsID: e.DimensionsID,
```

Comment:
This should be dimensionIDs not dimensionsID

Reply:
done